Formula/heksa.rb

```diff
@@ -0,0 +1,29 @@
+class Heksa < Formula
+  desc "CLI hex dumper with colors"
+  homepage "https://github.com/raspi/heksa"
+  url "https://github.com/raspi/heksa.git",
+      :tag      => "v1.12.1",
+      :revision => "54a68a2910a87b8b9cfcda28b8693e18dd9e2463"
+  head "https://github.com/raspi/heksa.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "bin/heksa"
+  end
+
+  test do
+    require "pty"
+
+    cp test_fixtures("test.png"), testpath/"test.png"
```

Comment:
Interesting, why do we need this step?

Reply:
To make command shorter 😀
It probably isn't needed, that's leftover from my previous experimentation and I'm lazy enough to leave it there 😀

Reply:
😆